docs/01-overview/main-areas/observability/README.md

```diff
@@ -6,8 +6,8 @@ title: What is Observability in Kyma?
 
 Kyma comes with tools that give you the most accurate and up-to-date monitoring, logging and tracing data.
 
-- [Prometheus](https://prometheus.io/) is the open source monitoring and alerting toolkit that provides the telemetry data. This data is consumed by different addons, including [Grafana](https://grafana.com/) for analytics and monitoring, and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) for handling alerts.
-- For logging, Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like log management system.
+- [Prometheus](https://prometheus.io/) is the open source monitoring and alerting toolkit that provides the telemetry data. This data is consumed by different addons, including [Grafana](https://grafana.com/) for analytics and monitoring, and [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) for handling alerts. By default, Prometheus stores up to 15 GB of data for a maximum period of 30 days. If the default size or time is exceeded, the oldest metrics are removed first.
+- For logging, Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like log management system. By default, Loki stores up to 30 GB of data for a maximum of 5 days, with maximum ingestion rate of 3 MB/s. If the default size or time is exceeded, the oldest logs are removed first.
```

Comment:
If the size is exhausted, Loki will crash. For now there is no real retention on size, only on time. Strategy is to expand the volume if it is running out of space

Reply:
Oh! That was taken from SKR docs. Then we need to improve it there too.

Reply:
Does Loki recover automatically after the crash, or is there need for another troubleshooting guide "Loki crashed" ?

Reply:
removed "size or", keeping only "time" for now.